internal/featuresupport/feature_support_test.go

```diff
@@ -239,6 +239,48 @@ var _ = Describe("V2ListFeatureSupportLevels API", func() {
 				false,
 			),
 		)
+
+		DescribeTable(
+			"test feature architecture support",
+			func(arch string, result bool) {
+				Expect(
+					isFeatureCompatibleWithArchitecture(
+						&NonStandardHAControlPlane{},
+						common.MinimumVersionForNonStandardHAOCPControlPlane,
+						arch,
+					),
+				).To(Equal(result))
+			},
+			Entry(
+				common.X86CPUArchitecture,
+				common.X86CPUArchitecture,
```

Comment:
Why the string is passed twice?

Reply:
Because you have to provide a description in a `DescribeTable` for each `Entry` (first parameter)